docs/data-management.rst

```diff
@@ -130,6 +130,33 @@ Browsing and rendering
 
 #. You can revert to the original settings for an Image or Dataset. For example, using the context menu for a Dataset in the tree, select ``Rendering Settings > Set Imported and Save``.
 
+
+Manage Images in Dataset/Projects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can organize the data in the left-hand side tree by creating new Projects, Datasets and linking the Images to those new containers.
+
+#. Use the Project |image2b| and Dataset |image2c| icons above the left hand side tree or the right-click contextual menu to create new Datasets or Projects.
+
+#. Drag and drop Images or Datasets between Datasets or Projects.
```

Comment:
Maybe: "Drag and drop Images between Datasets and Datasets between Projects" ?

Reply:
Fixed in https://github.com/ome/omero-guide-introduction/pull/5/commits/6d20ee1b14a66f1cbd81cbd7d6c2bcf4d7392ed4